doc/project/project_roles.rst

```diff
@@ -107,7 +107,9 @@ in addition to those listed for Contributors and Collaborators:
 * Right to set the overall architecture of the relevant subsystems or areas
   of involvement.
 * Right to make decisions in the relevant subsystems or areas of involvement,
-  in conjunction with the collaborators.
+  in conjunction with the collaborators. In cases of disagreement between
+  maintainers and other contributors, the maintainer's decision shall stand
+  unless overruled by the TSC.
```

Comment:
ok, this is covered down [below](https://docs.zephyrproject.org/latest/project/project_roles.html#escalation-process) in the same document, IMHO we should link to the escalation process here for clarity on how to get to the TSC.

Reply:
Changing to:

```
* Right to make decisions in the relevant subsystems or areas of involvement,
  in conjunction with the collaborators and submitters.
  See :ref:`escalation-process`.
```